ec2-hibinit-agent.spec

```diff
@@ -7,10 +7,13 @@ Group:          System Environment/Daemons
 License:        Apache 2.0
 Source0:        ec2-hibinit-agent-%{version}.tar.gz
 BuildArch:      noarch
-BuildRequires:  python2 python2-devel systemd acpid
+BuildRequires:  python2-devel
```

Comment:
Any reason why they are added in separate lines ? Also do we not require acpid anymore ?

Reply:
acpid is not required to build our service (The difference between `buildRequires` & `requires`)

The multi-line convention is a recommended convention to make our the spec file more legible in future (e.g. we end up adding many requirements)